autofs-5.0.8 - fix WITH_LIBTIRPC function name

From: Ian Kent <ikent@redhat.com>

autoconf function names AM_<name> are reserved so don't use them.
---
 CHANGELOG    |    1 +
 aclocal.m4   |    2 +-
 configure.in |    2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index 565153d..68db340 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -4,6 +4,7 @@
 - fix master map type check.
 - fix task manager not getting signaled.
 - allow --with-systemd to take a path arg.
+- fix WITH_LIBTIRPC function name.
 
 17/10/2013 autofs-5.0.8
 =======================
diff --git a/aclocal.m4 b/aclocal.m4
index 105e3e9..2115204 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -441,7 +441,7 @@ CFLAGS="$af_check_libtirpc_save_cflags"
 LDFLAGS="$af_check_libtirpc_save_ldflags"
 ])
 
-AC_DEFUN([AM_WITH_LIBTIRPC],
+AC_DEFUN([AF_WITH_LIBTIRPC],
 [AC_MSG_CHECKING([if libtirpc is requested and available])
 AC_ARG_WITH(libtirpc,
 [  --with-libtirpc         use libtirpc if available],
diff --git a/configure.in b/configure.in
index 5f29163..1eeb8dc 100644
--- a/configure.in
+++ b/configure.in
@@ -124,7 +124,7 @@ AC_SUBST(flagdir)
 #
 # Use libtirpc
 #
-AM_WITH_LIBTIRPC()
+AF_WITH_LIBTIRPC()
 AC_SUBST(TIRPCLIB)
 
 #
